cmd/antrea-agent/agent.go

```diff
@@ -543,6 +543,7 @@ func run(o *Options) error {
 	}
 	var l7FlowExporterController *l7flowexporter.L7FlowExporterController
 	if l7FlowExporterEnabled {
+		klog.InfoS("Feature gate L7FlowExporter is deprecated and will be removed in the future release")
```

Comment:
The library will already log something, so I think we can skip that one: https://github.com/kubernetes/component-base/blob/2fbd958bc1f70556c227642c181065e1ab195b2c/featuregate/feature_gate.go#L504